desktop/cmp/rest/RestGridModel.js

```diff
@@ -157,7 +156,7 @@ export class RestGridModel {
     }
 
     /** Load the underlying store. */
-    doLoadAsync(loadSpec) {
+    async doLoadAsync(loadSpec) {
```

Comment:
Good catch! This looks like something for which we could write a custom linting function, doesn't it?